projects/components/facet/facet.service.ts

```diff
@@ -45,14 +45,16 @@ export const enum FacetEventType {
     ClearFilters = "Facet_ClearFilters",
     AddFilter = "Facet_AddFilter",
     RemoveFilter = "Facet_RemoveFilter",
-    Open = "Facet_TreeOpen"
+    Open = "Facet_TreeOpen",
+    LoadMore = "Load_More"
 }
 
 // Types of events triggering a change event
 export const FACET_CHANGE_EVENTS = [
     FacetEventType.Loaded,
     FacetEventType.Add,
-    FacetEventType.Remove
+    FacetEventType.Remove,
+    FacetEventType.LoadMore
```

Comment:
I think LoadMore should not be in the "change" event, because there's no change in the data stored in user settings (it's just an internal change of the facet state). However, it seems the change events are missing AddAll and RemoveAll.

Reply:
Sorry I'm only noticing the answer now. I'll revert the commit then!